cruise/CruiseInformation.csv

```diff
@@ -175,6 +175,6 @@ KP20190808,R/V Kalipi,2019-08-08T13:00:00,2019-08-08T23:14:00,Endurance glider r
 EK20190813,R/V Elakha,2019-08-13T15:00:00,2019-08-13T23:00:00,Endurance ce02shsp_00015 deployment
 EK20190822,R/V Elakha,2019-08-22T12:00:00,2019-08-22T20:00:00,Endurance ce01issp_00014 and glider deployment
 RC20190827,R/V Rachel Carson,2019-08-27T12:00:00,2019-08-27T19:00:00,ce06issp_00011 deployment
-SKQ201920S,R/V Sikuliaq,2019-09-18T00:00:00,2019-10-05T23:59:00,Papa 7
+SKQ201920S,R/V Sikuliaq,2019-09-19T16:10:00,2019-10-05T00:28:00,Papa 7
 AR-38,R/V Neil Armstrong,2019-09-20T14:00:00,2019-09-22T15:15:00,Pioneer glider recovery
-AR-39,R/V Neil Armstrong,2019-09-25T00:00:00,2019-10-15T23:59:00,Pioneer 13
+AR-39,R/V Neil Armstrong,2019-09-25T16:55:00,2019-10-15T16:26:00,Pioneer 13
```

Comment:
Based on the image Sheri shared, shouldn't the depart time be 16:53?

Reply:
My notebook said 16:53, but the Elog said 16:55 and we are going to go with the ELog as the official start time.